<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerStructureEnhancerModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerStructureEnhancerModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Inputs</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="6" column="0" colspan="2">
          <widget class="QGroupBox" name="weightsGroupBox">
           <property name="title">
            <string>Weights</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Alpha:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="alphaDoubleSpinBox">
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
              <property name="value">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Beta:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="betaDoubleSpinBox">
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
              <property name="value">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Gamma:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="gammaDoubleSpinBox">
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
           <zorder>label_3</zorder>
           <zorder>label_2</zorder>
           <zorder>label_4</zorder>
           <zorder>alphaDoubleSpinBox</zorder>
           <zorder>betaDoubleSpinBox</zorder>
           <zorder>gammaDoubleSpinBox</zorder>
           <zorder>horizontalSpacer_2</zorder>
           <zorder>horizontalSpacer_3</zorder>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Input Volume:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="qMRMLNodeComboBox" name="inputVolMRMLNodeComboBox"/>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QGroupBox" name="structureGroupBox">
           <property name="title">
            <string>Structure</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QRadioButton" name="sheetRadioButton">
              <property name="text">
               <string>Sheet</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="tubeRadioButton">
              <property name="text">
               <string>Tube</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="blobRadioButton">
              <property name="text">
               <string>Blob</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Scale</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="minLabel">
              <property name="text">
               <string>Min:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="minDoubleSpinBox">
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="maxLabel">
              <property name="text">
               <string>Max:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="maxDoubleSpinBox">
              <property name="value">
               <double>2.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Scale Steps:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="stepSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Extras</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="brightCheckBox">
              <property name="text">
               <string>Bright Object</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="enhanceButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <slots>
    <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
    <slot>setMRMLScene(vtkMRMLScene*)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
   <slots>
    <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerStructureEnhancerModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputVolMRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enhanceButton</sender>
   <signal>clicked()</signal>
   <receiver>qSlicerStructureEnhancerModuleWidget</receiver>
   <slot>startEnhancement()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startEnhancement()</slot>
 </slots>
</ui>
